.devcontainer/README.md

```diff
@@ -33,15 +33,15 @@ The dev enviroment comes with some scripts to make this easier, They are located
 
 There are also VSCode tasks provided should you desire to use those. 
 
-The task BuildInit will update submodules and configure for openocd
+The task "update submodules" will update the git submodules
 
 
 
 ### Build
 
 You can use the build.sh script located in /opt/
 
-There will also eventually be a Build task. - Not written yet
+CMake is also configured and controlls for the CMake plugin are available in VSCode
```

Comment:
Typo

Reply:
<3 thank you!

Reply:
Fixed a whole bunch that my non tired eyes saw this morning, Thanks!